.github/workflows/dataflow_engine_chaos.yaml

```diff
@@ -221,14 +221,14 @@ jobs:
         run: |
           # Add a delay if test fails, to check whether the cluster can recover after chaos is removed
           sleep 60
-          kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "server-master"|xargs -I{} kubectl exec -it {} -- wget http://127.0.0.1:10240/debug/pprof/goroutine?debug=2 -O /log/{}_goroutines.log || true
-          kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "executor"|xargs -I{} kubectl exec -it {} -- wget http://127.0.0.1:10241/debug/pprof/goroutine?debug=2 -O /log/{}_goroutines.log || true
+          kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "server-master"|xargs -I{} kubectl exec -i -c server-master {} -- wget http://127.0.0.1:10240/debug/pprof/goroutine?debug=2 -O /log/{}_goroutines.log || true
```

Comment:
Can we rename the `server-master` to `master` and make the names symetric?

Reply:
Agreed, added an issue to track the rename. https://github.com/pingcap/tiflow/issues/6997